library/src/main/java/com/novoda/downloadmanager/NotificationChannelCreator.java

```diff
@@ -0,0 +1,30 @@
+package com.novoda.downloadmanager;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.content.Context;
+import android.os.Build;
+
+public class NotificationChannelCreator {
```

Comment:
@zegnus I've exposed this class so that the `custom` implementations can still use a notification channel without having to essentially duplicate the code here. Can hide behind an interface if you would prefer, that way clients can decide the priority etc of the channel.